compiler/AST/build.cpp

```diff
@@ -806,8 +805,7 @@ buildExternBlockStmt(const char* c_code) {
   BlockStmt* ret = buildChapelStmt(useBlock);
 
   // Check that the compiler supports extern blocks
-  // but skip these checks for chpldoc.
-  if (fDocs == false) {
+  {
```

Comment:
I think stray curly braces like this should be removed and the code unindented.  It implies a code structure that is not true and could be confusing